### PR TITLE
Architecture Diagram Spine 

### PR DESCRIPTION
**Pull request by meizabel**
Working on issue #126 
Concepts structure layout updated.
New term entries added.

### CODE REVIEW

File: _includes/doc-side-concepts-nav.html [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
+            <li><a href="{{ site.baseurl }}/docs/concepts/command-bus.html" {% if current[3] == 'command-bus.html' %}class='current'{% endif %}>Command Bus</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/command-store.html" {% if current[3] == 'command-store.html' %}class='current'{% endif %}>Command Store</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/event-bus.html" {% if current[3] == 'event-bus.html' %}class='current'{% endif %}>Event Bus</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/query-service.html" {% if current[3] == 'query-service.html' %}class='current'{% endif %}>Query Service</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/integration-event.html" {% if current[3] == 'integration-event.html' %}class='current'{% endif %}>Integration Event</a></li>
+            <li><a href="{{ site.baseurl }}/docs/concepts/command-service.html" {% if current[3] == 'command-service.html' %}class='current'{% endif %}>Command Service</a></li>
```
Comment (alexander-yevsyukov) [inline _includes/doc-side-concepts-nav.html]: Please reorder the items:
1. Bounded Context
2. Integration Event
3. Command Bus
4. Command Store
5. Command Service
6. Event Bus
7. Event Store
8. Aggregate Mirror
9. Query Service

File: docs/concepts/command-service.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: Command Service
+headline: Command Service
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Command Service</h2> 
+
+Commands service concept to be added.
```
Comment (alexander-yevsyukov) [inline docs/concepts/command-service.md]: Have such TODO notes in a special style. It should be clearly visible that it's not a regular text.

File: docs/concepts/command-service.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+
```
Comment (alexander-yevsyukov) [inline docs/concepts/command-service.md]: Please also do not add empty lines. They will add up to the difference in the next update.

File: docs/concepts/index.md [hunk 4]
```diff
@@ -11,9 +11,20 @@ sidenav: doc-side-concepts-nav.html
...
 
-Below you can find a typical Spine Event Engine application architecture; its building blocks are also described below.
+Here is a basic view of Spine Event Engine application architecture.
+
+<img src="{{ site.baseurl }}/img/concepts/spine-architecture-diagram-1.0.0.png" alt="Spine Event Engine Architecture">
+
+These are the parts of Spine architecture you are going to interact with when working with Spine. 
+
+<img src="{{ site.baseurl }}/img/concepts/spine-architecture-diagram-1.0.0-faded.png" alt="Spine Event Engine Architecture">
+
```
Comment (alexander-yevsyukov) [inline docs/concepts/index.md]: Please remove empty lines.

File: docs/concepts/integration-event.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+title: Integration Event
+headline: Integration Event
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Integration Event</h2> 
+
+Integration Event concept to be added.
```
Comment (alexander-yevsyukov) [inline docs/concepts/integration-event.md]: Please have a style for the TODO text.

File: docs/concepts/index.md [hunk 6]
```diff
@@ -11,9 +11,20 @@ sidenav: doc-side-concepts-nav.html
 <div id="toc" class="toc hide-block"></div>
 This section introduces key concepts of Spine framework and its parts, helps you get a deeper understanding of how it works and provides some usage notes when creating Spine-based cloud application. 
 
-We assume that you are familiar with the basics of Domain-Driven Design. 
+<p class="note">We assume that you are familiar with the basics of Domain-Driven Design.</p> 
 
-Below you can find a typical Spine Event Engine application architecture; its building blocks are also described below.
+Here is a basic view of Spine Event Engine application architecture.
```
Comment (alexander-yevsyukov) [inline docs/concepts/index.md]: I'm not sure about this wording. То, что мы тут показываем, это — принципиальная схема архитектуры приложения. Сейчас текст ”Spine Event Engine application” звучит как-будто у самого framework есть какое-то приложение.

File: docs/concepts/command-service.md (hunk 2 above)
Comment (meizabel) [reply docs/concepts/command-service.md]: LGTM 
Agreed on using <p class="coming-soon">Coming soon</p> formatting style with web designer.

**Pull request merged**